packages/manager/modules/vps/src/backup-storage/add/add-backup-storage.component.js

```diff
@@ -0,0 +1,11 @@
+import controller from './add-backup-storage.controller';
+import template from './add-backup-storage.html';
+
+export default {
+  bindings: {
+    goBackToDetails: '<',
```

Comment:
```suggestion
    goBack: '<',
```
This way, the day we change where the component is, you don't have to rename this ;)